schemas/list-workertypes-response.yml

```diff
@@ -20,10 +20,45 @@ properties:
           minLength:    {$const: identifier-min-length}
           maxLength:    {$const: identifier-max-length}
           pattern:      {$const: identifier-pattern}
+        provisionerId:
+          title:        "Provisioner ID"
+          type:         string
+          minLength:    {$const: identifier-min-length}
+          maxLength:    {$const: identifier-max-length}
+          pattern:      {$const: identifier-pattern}
+        stability:
+          title:        "Stability"
+          description: |
+            The stability of the worker-type. Accepted values:
+            'experimental|stable|deprecated'
+          type:         string
+          enum:         ["experimental", "stable", "deprecated"]
+        description:
+          title:        "Description"
+          description: |
+            Description of the worker-type.
+          type:         string
+        documentation:
```

Comment:
Yes, that's what i meant, although I think the paths are actually
```
GET /v1/provisioners/:provisionerId/worker-types  # list of workerTypes without details
GET /v1/provisioners/:provisionerId/worker-types/:workerType  # single workerType with details
```

so from a REST perspective, those are the same thing, just one view has more detail than the other.